api/apps/api/src/modules/geo-feature-tags/geo-feature-tags.service.ts

```diff
@@ -109,4 +116,30 @@ export class GeoFeatureTagsService {
 
     return right(true);
   }
+
+  async deleteTagForProject(
+    userId: string,
+    projectId: string,
+    tag: string,
+  ): Promise<Either<typeof projectNotEditable | typeof projectNotFound, any>> {
```

Comment:
tiny: I think you can avoid this `any` in the right side of the `Either`

Reply:
you mean as in using boolean/true/void/whatever? if so, yeah, I can modify that on another commit